public/app/features/plugins/sandbox/distortion_map.ts

```diff
@@ -356,6 +371,19 @@ function distortAppend(distortions: DistortionMap) {
           return document.createDocumentFragment();
         }
       }
+      // if the node is a script, load it into the sandbox
+      // this allows webpack chunks to be loaded into the sandbox
+      // loadScriptIntoSandbox has restrictions on what scripts can be loaded
+      if (sandboxEnv && arg && nodeType === 'script' && arg instanceof HTMLScriptElement) {
```

Comment:
even though these checks look redundant they are necessary for typescript.

Reply:
I'd stuff this logic in a var like `isScriptTag` to make it easier to scan the code.